### PR TITLE
Merge register extra content

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd

### CODE REVIEW

File: lagunita/lms/templates/student_account/register-extra.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<%namespace name='static' file='static_content.html'/>
```
Comment (stvstnfrd) [inline lagunita/lms/templates/student_account/register-extra.html]: Why do we need static here but no on CME?

File: cme/lms/templates/student_account/register-extra.html [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<%!
+    from django.utils.translation import ugettext as _
+%>
+<fieldset>
+    <legend>&dagger; Medical Designation or Professional Title</legend>
```
Comment (stvstnfrd) [inline cme/lms/templates/student_account/register-extra.html:5]: This is how it was previously, but should we just go ahead and i18n this?

Comment (caesar2164) [reply cme/lms/templates/student_account/register-extra.html:5]: I think we want it exactly as written, so no i18n...

Comment (stvstnfrd) [reply cme/lms/templates/student_account/register-extra.html:5]: It's not `i18n`'ed elsewhere, so that's fine.

**Comment by stvstnfrd** [comment]
+1 pending tests

**Pull request merged**